JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -91,185 +99,274 @@
 
-    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany#hasDesignCapacity">
-        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany#Factory"/>
-        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany#DesignCapacity"/>
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasDesignCapacity">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/DesignCapacity"/>
         <rdfs:comment>A relationship between a factory and its design capacity.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
-        <rdfs:label>has design capacity</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>design capacity</rdfs:label>
     </owl:ObjectProperty>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#hasGeneratedHeat -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasEnergyConsumptionPerUnitFloorArea -->
 
-    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany#hasGeneratedHeat">
-        <rdfs:domain>
-            <owl:Class>
-                <owl:unionOf rdf:parseType="Collection">
-                    <rdf:Description rdf:about="http://www.purl.org/oema/infrastructure/Building"/>
-                    <rdf:Description rdf:about="http://www.theworldavatar.com/ontology/ontocape/chemicalprocesssystem/cpsrealization/plant/PlantItem"/>
-                </owl:unionOf>
-            </owl:Class>
-        </rdfs:domain>
-        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany#GeneratedHeat"/>
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasEnergyConsumptionPerUnitFloorArea">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/EnergyConsumptionPerUnitFloorArea"/>
+        <rdfs:comment>A relationship between a factory and the amount of energy used per unit floor area.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Energy Consumption Per Unit Floor Area</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasFloorArea -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasFloorArea">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacility"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/FloorArea"/>
+        <rdfs:comment>A relation between an industrial facility and its gross floor area.</rdfs:comment>
```

Comment:
Is it gross floor area? or just the floor area i.e. the Lod0footprint or equivalent?

Reply:
It is the gross floor area, not the Lod0footprint. The gross floor area is the one relevant for calculating heat emissions of data centres and precision engineering plants.

Reply:
In that case it makes sense to rename the concept to "GrossFloorArea" to avoid confusion.

@YiKaiTsai Is there a gross floor area concept from OntoBuiltEnv/other ontologies that we could reuse?